WcaOnRails/app/models/user_group.rb

```diff
@@ -9,6 +9,14 @@ class UserGroup < ApplicationRecord
     translators: "translators",
   }
 
+  def self.group_types_with_status_metadata
```

Comment:
If I'm not mistaken, this returns just an array of strings.
Why is it called `...with_status_metadata`?

Reply:
These will return only those groups which have `status` metadata, and not the ones that doesn't have `status` metadata like `translators`, `delegate_probations`, etc.

Reply:
Ah, in that case I misunderstood. To me, the function name sounds like "return some group types and also return their status metadata". Instead, it should be like "return only those group types which can have status metadata".

Please spend a brief moment to see if you can come up with a more concise name, but don't waste more than 2 minutes on this. In case of doubt, it's fine the way it currently is :smile:

Reply:
Okay, in two minutes I got another name: `group_types_containing_status_metadata`. Pushed it, is this fine?